Swift/BRCryptoDemo/CoreDemoAppDelegate.swift

```diff
@@ -105,18 +124,7 @@ class CoreDemoAppDelegate: UIResponder, UIApplicationDelegate, UISplitViewContro
         let subscription = BlockChainDB.Subscription (id: subscriptionId, endpoint: nil);
         self.system.subscribe (using: subscription)
 
-        var networksNeeded: [String] = []
-
-        #if TESTNET
-        networksNeeded += ["bitcoin-testnet"] //, "ripple-testnet"] // ...
-        #endif
-
-        #if MAINNET
-        networksNeeded += ["bitcoin-mainnet"] // ...
-        #endif
-
-        print ("APP: Networks          : \(networksNeeded)")
-        self.system.start (networksNeeded: networksNeeded) //, */"ethereum-mainnet"])
+        self.system.configure()
```

Comment:
Food for thought, but do we want to enforce that configure has been run once (and only once)?